### PR TITLE
Add a new inSubset ROBOT template

### PR DESCRIPTION
**Pull request by rays22**
This commit intends to add a new `inSubset` ROBOT template.
This commit is related to https://github.com/obophenotype/uberon/issues/2127#issuecomment-979791339 .

### CODE REVIEW

**Comment by dosumis** [comment]
I think this would work nicely.  We need to delete the example before merging though.  Also @nicolevasilevsky what AP are we now using for linking to tickets dc:source or rdfs:seeAlso ?  (seems like something should have in our new centralised editor's guide).

File: src/templates/in-subset.template.tsv [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Class ID	Subset tag	PROVENANCE
+ID	AI oboInOwl:inSubset	>AI oboInOwl:source
+EXAMPLE:0000000	never_in_euarchontoglires	https://github.com/obophenotype/uberon/issues/????
```
Comment (dosumis) [inline src/templates/in-subset.template.tsv:3]: Delete example line.

**Review by dosumis — COMMENTED** [review]
Activating template will also require adding OWL file it produces to components + import statement on uberon-edit.owl

**Comment by matentzn** [comment]
@rays22 can you finish and merge this PR?

File: src/templates/in-subset.template.tsv (hunk 1 above)
Comment (rays22) [reply src/templates/in-subset.template.tsv]: Done.

**Comment by rays22** [comment]
> Activating template will also require adding OWL file it produces to components + import statement on uberon-edit.owl

@dosumis ,
Do I need to do that before merging?

**Comment by matentzn** [comment]
Best put this PR on our meeting agenda and I will tell you how to create a ROBOT template component with ODK!

**Comment by nicolevasilevsky** [comment]
we use rdfs:seeAlso in Mondo

**Comment by shawntanzk** [comment]
1) Create new component 
2) Have component dependent on tsv template 
3) component = template cmd that turns template into owl 

@shawntanzk  to point @rays22 to docs and fix up docs whie at it

**Comment by shawntanzk** [comment]
Currently doesnt seem like there is documentation on components - @shawntanzk to work on documentation for this

**Comment by shawntanzk** [comment]
docs: https://github.com/obophenotype/uberon/pull/2339

**Comment by rays22** [comment]
> linking to tickets dc:source or rdfs:seeAlso

FYI: This template uses `oio:source` instead of `rdfs:seeAlso` for provenance **at the annotation level** as discussed here:
<https://github.com/obophenotype/uberon/pull/2108#issuecomment-947621119>

**Comment by shawntanzk** [comment]
@rays22 I'm a bit confused, the tsv you added is for inSubset, but you activated in-taxon template?

File: src/ontology/uberon-odk.yaml [hunk 2]
```diff
@@ -87,23 +87,24 @@ subset_group:
...
 owltools_memory: '20G'
 create_obo_metadata: FALSE
 documentation:
   documentation_system: mkdocs
 components:
   products:
-    - filename: mappings.owl
\ No newline at end of file
+    - filename: mappings.owl
+    - filename: in-taxon.owl
```
Comment (shawntanzk) [inline src/ontology/uberon-odk.yaml:110]: should this be in-subset.owl?

Comment (rays22) [reply src/ontology/uberon-odk.yaml:110]: Yes, you are right.

Comment (rays22) [reply src/ontology/uberon-odk.yaml]: Fixed.

**Comment by shawntanzk** [comment]
@rays22 you need to run `sh run.sh update_repo` after the change in the yaml file

**Review by matentzn — APPROVED** [review]

File: src/ontology/components/in-taxon.owl [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
```
Comment (matentzn) [inline src/ontology/components/in-taxon.owl:1]: wrong name, should be `in-subset.owl`

File: docs/odk-workflows/RepositoryFileStructure.md [hunk 4]
```diff
@@ -53,3 +53,4 @@ These are the components in UBERON
 | Filename | URL |
 | -------- | --- |
 | mappings.owl | None |
+| in-taxon.owl | None |
```
Comment (matentzn) [inline docs/odk-workflows/RepositoryFileStructure.md:56]: is-subset.owl

**Review by matentzn — APPROVED** [review]

**Review by matentzn — CHANGES_REQUESTED** [review]
Sorry in-subset.owl not `in-taxon.owl`.

**Review by matentzn — APPROVED** [review]

**Pull request merged**